include/clover_field_order.h

```diff
@@ -585,8 +587,9 @@ namespace quda {
         const AllocInt offset; // offset can be 32-bit or 64-bit
         const int volumeCB;
 
-	const bool twisted;
-	const real mu2;
+        const QudaTwistFlavorType twist_flavor;
+        const real mu2;
```

Comment:
All of these parameters --- `mu2`, `epsilon2`, `rho`, as well as `a`, `b`, `c` further below --- have gotten confusing. Can these please be documented on the wiki in some fashion?